packages/fhir-converter/src/templates/cda/Resources/DiagnosticReport.hbs

```diff
@@ -36,7 +36,11 @@
             {{/each}}
         ],
         "status":{{>ValueSet/DiagnosticReportStatus.hbs code=diagReport.statusCode.code}},
-        "code":{{>DataType/CodeableConcept.hbs code=diagReport.code}},
+        {{#if diagReport.code.code}}
```

Comment:
bug? code.code
Looks wrong since within a code, the code might be in the translation etc.